packages/frontend/pages/MessageBoard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect} from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export function Message( {message} ) {
+    return (
+        <div>
+            <h4>Name: {message.name}</h4>
+            <p>Message:</p>
+            <p>{message.msg}</p>
+            <p>[DEBUG]MessageID: {message.id}</p>
+            <br></br>
+        </div>
+    )
+}
+
+export function MessageList({ messages }) {
+    return (
+        messages.map(message => {
+            return <Message key={message.id} message={message}/>
+        })
+    )
+}
+
+export default function MessageBoard() {
+    const [messages, setMessages] = useState([])
+    const messageNameRef = React.useRef<HTMLInputElement>(null)
+    const messageMsgRef = React.useRef<HTMLTextAreaElement>(null)
+
+    function uploadMessage(e){
+        const messageName:String = messageNameRef.current.value
+        const messageMsg:String = messageMsgRef.current.value
+        if(messageName === "" || messageMsg === "") return
+        setMessages(prevMessages => {
+            return [...prevMessages, {id:uuidv4(), name:messageName, msg:messageMsg}]
```

Comment:
Can be made inline (see comment in `MessageList`)

Reply:
fixed